libs/muddle/src/peer_selector.cpp

```diff
@@ -41,12 +41,12 @@ namespace {
 using namespace std::chrono;
 using namespace std::chrono_literals;
 
-constexpr auto        MIN_ANNOUNCEMENT_INTERVAL = 15min;
-constexpr auto        MAX_ANNOUNCEMENT_INTERVAL = 30min;
-constexpr std::size_t MINIMUM_PEERS             = 3;
+constexpr auto        MIN_ANNOUNCEMENT_INTERVAL = 10s;
+constexpr auto        MAX_ANNOUNCEMENT_INTERVAL = 30s;
```

Comment:
Are these purposefully set down to 10s and 30s or did you just foget to set them up again? Long term I think we should move them into some config file.

Reply:
Yes - this was an intentional change to seem up network convergence